theme/CNAME

```diff
@@ -1 +0,0 @@
-melange.re
```

Comment:
This are not copied by `mike` so added them in the `gh-pages` branch manually: https://github.com/melange-re/melange-re.github.io/commit/42b5b09021041cc312f389afc770f7a5b82c1389